src/views/Orders.vue

```diff
@@ -313,4 +318,10 @@ export default defineComponent({
   align-items: flex-end;
   row-gap: 4px;
 }
+
+@media (min-width: 991px){
+  .header{
```

Comment:
```suggestion
  .find{
```

Reply:
Yes sir, Improved the class name from header to find.

Reply:
I have removed header class and used css selector.